src/axom/sidre/docs/sphinx/view.rst

```diff
@@ -9,113 +9,117 @@
 View
 ==========
 
-A Sidre View describes data and provides access to it. A View can describe
-a (portion) of a data allocation in **any way that is compatible** with the
-allocation. Specifically, the allocation must contain enough bytes to support
-the description. In particular, the data type of a View description need
-not match the types associated with the allocation employed by other Views
-into that data.
+A Sidre ``View`` objects describes data and provides access to it. A view can 
+describe a (portion) of a data allocation in **any way that is compatible** 
+with the allocation. Specifically, the allocation must contain enough bytes 
+to support the description. In particular, the data type of a view description 
+need not match the types associated with the allocation or types of other views
+that refer to that data.
 
-.. note:: View objects can only be created and destroyed using Group methods 
-          provided for this. The View constructor and destructor are private.
+.. note:: ``View`` objects can only be created and destroyed using ``Group``
+          class methods. The ``View`` class constructor and destructor are 
+          private.
 
-Each View object has a name and is owned by one Group in a Sidre Group 
-hierarchy; its *owning* Group. A View maintains a pointer to the Group that 
-owns it.
+Each view has a name and is owned by one group in a Sidre group hierarchy; i.e.,
+its *owning* Group. A view maintains a pointer to the group that owns it.
```

Comment:
Perhaps change "owning Group" to "owning group" to match the capitalization earlier in the sentence.  This capitalization fix may need to propagate throughout the file (such as line 27).

Reply:
Good catch. fixed.